src/modules/apps/e2e/specsState.ts

```diff
@@ -0,0 +1,36 @@
+import { SpecReport, AppReport, TestReport } from 'vtex'
```

Comment:
These types are specific to testing logic - it should be removed from 'vtex' and moved to here